pkg/codegen/internal/test/testdata/aws-s3-logging.pp.ts

```diff
@@ -0,0 +1,8 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+const logs = new aws.s3.Bucket("logs", {});
+const bucket = new aws.s3.Bucket("bucket", {loggings: [{
+    targetBucket: logs.bucket,
+}]});
+export const targetBucket = bucket.loggings.apply(loggings => loggings[0].targetBucket);
```

Comment:
Should this output be lifted?

Reply:
Yup, filed https://github.com/pulumi/pulumi/issues/4465